rskj-core/src/main/java/co/rsk/peg/utils/BridgeEventLoggerImpl.java

```diff
@@ -245,6 +246,21 @@ public void logPegoutConfirmed(Sha256Hash btcTxHash, long pegoutCreationRskBlock
         this.logs.add(new LogInfo(BRIDGE_CONTRACT_ADDRESS, encodedTopics, encodedData));
     }
 
+    @Override
+    public void logPegoutTransactionCreated(Sha256Hash btcTxHash, List<Coin> outpointValues) {
+        if (btcTxHash == null){
+            throw new IllegalArgumentException("btcTxHash param cannot be null");
```

Comment:
`"[logPegoutTransactionCreated] btc transaction hash cannot be null"`

Reply:
That's for logs. This is just an exception.

Reply:
you are right. My bad :)